x/auctions/keeper/grpc_query_test.go

```diff
@@ -0,0 +1,153 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package keeper_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+
+	testutil "github.com/evmos/evmos/v19/testutil"
+	testnetwork "github.com/evmos/evmos/v19/testutil/integration/evmos/network"
+	testutiltx "github.com/evmos/evmos/v19/testutil/tx"
+	"github.com/evmos/evmos/v19/utils"
+	"github.com/evmos/evmos/v19/x/auctions/types"
+)
+
+func TestAuctionInfo(t *testing.T) {
+	var network *testnetwork.UnitTestNetwork
+	ibcCoin := sdk.NewCoin("ibc/D219F3A490310B65BDC312B5A644B0D56FFF1789D894B902A49FBF9D2F560B32", sdk.NewInt(1))
+	evmosCoin := sdk.NewCoin("aevmos", sdk.NewInt(1))
+	anotherNativeCoin := sdk.NewCoin("aeth", sdk.NewInt(1))
+	validBidderAddr, _ := testutiltx.NewAccAddressAndKey()
+
+	testCases := []struct {
+		name        string
+		malleate    func()
+		expResp     func() *types.QueryCurrentAuctionInfoResponse
+		expPass     bool
+		errContains string
+	}{
+		{
+			name: "pass with default genesis state",
+			malleate: func() {
+			},
+			expResp: func() *types.QueryCurrentAuctionInfoResponse {
+				return &types.QueryCurrentAuctionInfoResponse{
+					Tokens:        nil,
+					CurrentRound:  0,
+					HighestBid:    sdk.NewCoin(utils.BaseDenom, sdk.NewInt(0)),
+					BidderAddress: "",
+				}
+			},
```

Comment:
why is this a function and not just `types.QueryCurrentAuctionInfoResponse`?


```suggestion
			expResp: &types.QueryCurrentAuctionInfoResponse{
				Tokens:        nil,
				CurrentRound:  0,
				HighestBid:    sdk.NewCoin(utils.BaseDenom, sdk.NewInt(0)),
				BidderAddress: "",
			},
```

Reply:
Nice catch, I believe I used a func to leverage var defined inside the Run of the test but I refactored the code and now it is not needed anymore! Thanks!